Fixed build scripts broken by docker.

mtilbrook-dev opened the pull request:
These changes fix the client build scripts. But they break the docker… build and run.

I’ll try and find a way to mount all the app folder and have docker ignore the node_modules and elm-stuff

mtilbrook-dev committed:
These changes fix the client build scripts. But they break the docker build and run.

I’ll try and find a way to mount all the app folder and have docker ignore the node_modules and elm-stuff

marukami commented:
I'll need to find a way to share all the source files without sharing the node modules. #65 Needs Linux specific native modules and it can use the MacOS binaries.

mtilbrook-dev committed:
finished inital client docker image.

elm-live does not seems to work inside a container. But, you can still use the `yarn server` to compile and run a client.
